src/components/App/data.tsx

```diff
@@ -0,0 +1,102 @@
+export const dataset={"data":{
```

Comment:
good call faking the data so we don't need to switch all the API call to the fake json